qwed/guards/attestation_guard.py

```
@@ -2,15 +2,16 @@
 import hashlib
+import os
 from typing import Dict, Any, Optional
 
 class AttestationGuard:
     """
     Generates cryptographic proofs (JWTs) for verification results.
     Acts as a 'Digital Notary' for AI safety checks.
     """
-    def __init__(self, secret_key: str = "default-dev-secret-do-not-use-in-prod"):
-        self.secret = secret_key
+    def __init__(self, secret_key: str = None):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)